src/models/thelio-mira-b1.0/internal-overview.md

```diff
@@ -0,0 +1,33 @@
+# Thelio Mira (Internal Overview)
+
+## Side overview:
+
+![Internal overview (side)](./img/internal-overview.webp)
+
+- CPU cooler & CPU cooler fan are highlighted in solid green
+- CPU fan connectors are highlighted in solid pink
+- RAM slots are highlighted in dashed red
+- 2.5" SATA drive slot cover is highlighted in dashed pink
+- M.2 slots are highlighted in dashed cyan (located underneath the heatsinks)
+- PCIe slots are highlighted in yellow
```

Comment:
The PCIe 3.0 x4 slot is highlighted in Purple, should that be added to the list? 

I'm also not sure the top PCIe slot should be highlighted with dashed yellow if we aren't going to mention what dashed yellow means anywhere. Dashed vs solid lines of the same color refer to unrelated components in some cases, so I feel like it should be mentioned if it's going to be a different line style.

Reply:
> The PCIe 3.0 x4 slot is highlighted in Purple, should that be added to the list?

Yes, added that to the list and relabeled the existing entry.

> I'm also not sure the top PCIe slot should be highlighted with dashed yellow if we aren't going to mention what dashed yellow means anywhere. Dashed vs solid lines of the same color refer to unrelated components in some cases, so I feel like it should be mentioned if it's going to be a different line style.

The reason for dashes in that case was to signify that it's behind the GPU; it otherwise looks like we're drawing a box around nothing since the other two boxes have the slots visible. (For the Mega and Massive internal overviews, I removed the GPUs as well as the 2.5" drive covers for this photo, unlike the mira-r1 section, and I'm thinking it should probably go back to that style for future sections.)

I could just add "Top slot (dashed yellow) is behind the GPU", could change it back to solid yellow and move it up so it's at the top of the GPU (overlapping the top M.2 slot's box), or I could retake the photo without the GPU installed, depending on how severe we think this is.

I will point out that in any other case where dashed vs. solid matters, we don't just label the dashed color, we also refer to the solid color using the word "solid." This line says only "yellow" because it includes all stroke styles of yellow.

Reply:
>I could just add "Top slot (dashed yellow) is behind the GPU"
I like that the best. Fair point regarding "solid", but I do like it being a bit more explicit so nobody's left wondering.

Reply:
Done.